src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -59,7 +59,7 @@ public class CustomAsyncResult : IAsyncResult
     public static class Helpers
     {
         public static bool IsWindowsImplementation { get; } = PlatformDetection.IsWindows && PlatformDetection.IsNotOneCoreUAP;
-        public static bool IsNotWindowsImplementation => !IsWindowsImplementation;
+        public static bool IsNotWindowsImplementation => !IsWindowsImplementation && PlatformDetection.IsNotOneCoreUAP;
```

Comment:
IMO, I find this adding to the confusion. I think `IsWindowsImplementation` should return `true` for all Windows versions. Nano, UWP are both Windows implementations.
We should instead alter tests to clearly add what they work on and what they don't work on.